<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">
	<meta name="viewport"    content="width=device-width, initial-scale=1.0">
	<meta name="description" content="">
	<meta name="author"      content="Sergey Pozhilov (GetTemplate.com)">
	
	<title>Blog | Fan Feng Personal Website</title>

	<link rel="shortcut icon" href="assets/images/gt_favicon.png">
	
	<!-- Bootstrap -->
	<link href="https://netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.no-icons.min.css" rel="stylesheet">
	<!-- Icon font -->
	<link href="https://netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css" rel="stylesheet">
	<!-- Fonts -->
	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Alice|Open+Sans:400,300,700">
	<!-- Custom styles -->
	<link rel="stylesheet" href="assets/css/styles.css">

	<!--[if lt IE 9]> <script src="assets/js/html5shiv.js"></script> <![endif]-->
</head>
<body>

<header id="header">
	<div id="head" class="parallax" parallax-speed="2">
		<h1 id="logo" class="text-center">
			<img class="img-circle" src="assets/images/fan_portrait.jpg" alt="">
			<span class="title">Fan Feng (封凡)</span>
			<span class="tagline">
				Ph.D. candidate<br>
				Department of Computational Medicine and Bioinformatics, University of Michigan<br>
				密歇根大学 计算药物学与生物信息学系 博士生<br>
				<br>
				Alumni<br>
				College of Chemistry and Molecular Engineering, Peking University<br>
				北京大学 化学与分子工程学院 毕业生
			</span>
			<!--<a href="">fanfeng@umich.edu</a></span> -->
		</h1>
	</div>

	<nav class="navbar navbar-default navbar-sticky">
		<div class="container-fluid">

			<div class="navbar-header">
				<button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button>
			</div>

			<div class="navbar-collapse collapse">

				<ul class="nav navbar-nav">
					<li class="active"><a href="index.html">
						Home<br>
						主页
					</a></li>
					<li class="dropdown">
						<a href="#" class="dropdown-toggle" data-toggle="dropdown">
							Resume<br>
							简历
							<b class="caret"></b>
						</a>
						<ul class="dropdown-menu">
							<li><a href="cv_english.html">English Version 英文版</a></li>
							<li><a href="cv_chinese.html">Chinese Version 中文版</a></li>
						</ul>
					</li>
					<li><a href="about.html">
						Activities<br>
						动态
					</a></li>
					<li><a href="blog.html">
						Blog & Vlog<br>
						博客 & 视频
					</a></li>
				</ul>

			</div><!--/.nav-collapse -->
		</div>
	</nav>
</header>

<main id="main">

	<div class="container">
		<div class="row topspace">
			<div class="col-sm-10 col-sm-offset-1">
															
				<article class="post">
					<a id="genomickb"></a>
					<header class="entry-header">
						<div class="entry-meta"> 
							<span class="posted-on"><time class="entry-date published">April 6, 2023</time></span>
						</div>
						<h1 class="entry-title">Introducing GenomicKB!</h1>
					</header>
					<div class="entry-content">
						<p>
							A Knowledge Graph is a machine-readable format used to organize and represent information. It connects real-world concepts (displayed as nodes) and their relationships (displayed as edges) to link and integrate data from diverse sources.
							The advantages of using Knowledge Graphs include 1) data integration with flexibility and scalability, 2) supporting reasoning and inference, and 3) facilitating machine learning and AI.
						</p>
						<p>
							GenomicKB is a specialized Knowledge Graph that consolidates genomic data from over 30 consortia and portals, comprising 347 million genomic entities, 1.36 billion relations, and 3.9 billion entity and relation properties.
							It focuses on the human genome, epigenome, transcriptome, and 4D nucleome, and provides a coding-free and intuitive query system to turn multi-modal data analysis into a simple process.
						</p>
					</div>
					<div class="entry-content">
						<p><img alt="" src="assets/images/blog_picture/202212_GKB.png"></p>
					</div>
					<a href="https://gkb.dcmb.med.umich.edu" class="more-link">Link to the Website&#8230;</a><br>
				</article>

				<article class="post">
					<a id="2023rome"></a>
					<header class="entry-header">
						<div class="entry-meta">
							<span class="posted-on"><time class="entry-date published">March 17, 2023</time></span>
						</div>
						<h1 class="entry-title">My Travel in Rome, Italy!</h1>
					</header>
					<div class="entry-content">
						<p><img alt="" src="assets/images/blog_picture/202301_Rome.jpg"></p>
					</div>
					<a href="https://www.bilibili.com/video/BV19M411W75A/" class="more-link">Link to the Video&#8230;</a><br>
				</article>

				<article class="post">
					<a id="caesar"></a>
					<header class="entry-header">
						<div class="entry-meta">
							<span class="posted-on"><time class="entry-date published">April 30, 2022</time></span>
						</div>
						<h1 class="entry-title">Introducing CAESAR!</h1>
					</header>
					<div class="entry-content">
						<p>
							The resolution of chromatin conformation capture technologies keeps increasing, and the recent nucleosome resolution chromatin contact maps allow us to explore how fine-scale 3D chromatin organization is related to epigenomic states in human cells.
							Using publicly available Micro-C datasets, we develop a deep learning model, CAESAR, to learn a mapping function from epigenomic features to 3D chromatin organization. The model accurately predicts fine-scale structures, such as short-range chromatin loops and stripes, that Hi-C fails to detect. With existing epigenomic datasets from ENCODE and Roadmap Epigenomics Project, we successfully impute high-resolution 3D chromatin contact maps for 91 human tissues and cell lines. In the imputed high-resolution contact maps, we identify the spatial interactions between genes and their experimentally validated regulatory elements, demonstrating CAESAR’s potential in coupling transcriptional regulation with 3D chromatin organization at high resolution.
						</p>
					</div>
					<div class="entry-content">
						<p><img alt="" src="assets/images/blog_picture/202204_CAESAR.png"></p>
					</div>
					<a href="https://www.nature.com/articles/s41467-022-29695-6" class="more-link">Link to the Paper&#8230;</a><br>
				</article>

				<article class="post">
					<a id="schictools"></a>
					<header class="entry-header">
						<div class="entry-meta">
							<span class="posted-on"><time class="entry-date published">May 20, 2021</time></span>
						</div>
						<h1 class="entry-title">Introducing scHiCTools!</h1>
					</header>
					<div class="entry-content">
						<p>
							Single-cell Hi-C (scHi-C) sequencing technologies allow us to investigate three-dimensional chromatin organization at the single-cell level. However, we still need computational tools to deal with the sparsity of the contact maps from single cells and embed single cells in a lower-dimensional Euclidean space. This embedding helps us understand relationships between the cells in different dimensions, such as cell-cycle dynamics and cell differentiation. We present an open-source computational toolbox, scHiCTools, for analyzing single-cell Hi-C data comprehensively and efficiently. The toolbox provides two methods for screening single cells, three common methods for smoothing scHi-C data, three efficient methods for calculating the pairwise similarity of cells, three methods for embedding single cells, three methods for clustering cells, and a build-in function to visualize the cells embedding in a two-dimensional or three-dimensional plot. scHiCTools, written in Python3, is compatible with different platforms, including Linux, macOS, and Windows.
						</p>
					</div>
					<div class="entry-content">
						<p><img alt="" src="assets/images/blog_picture/202105_scHiCTools.PNG"></p>
					</div>
					<a href="https://github.com/liu-bioinfo-lab/scHiCTools" class="more-link">Link to the Tool&#8230;</a><br>
				</article>

			</div> 
		</div>

		<center class="">
			<ul class="pagination">
				<li class="disabled"><a href="">&laquo;</a></li>
				<li class="active"><a href="blog.html">1</a></li>
				<!--<li><a href="blog2.html">2</a></li>
				<li><a href="">3</a></li>
				<li><a href="">4</a></li>
				<li><a href="">5</a></li>
				<li><a href="">6</a></li>
				<li><a href="">&raquo;</a></li> -->
			</ul>
		</center>
	

	</div>	<!-- /container -->

</main>

<footer id="footer">
	<div class="container">
		<div class="row">
			<div class="col-md-3 widget">
				<h3 class="widget-title">
					Contact<br>
					联系我
				</h3>
				<div class="widget-body">
					<p>E-mail 邮箱:<br>
						<a href="mailto:#">fanfeng@umich.edu</a><br>
						<br>
						Address 地址:<br>
						Palmer Commons 2035, 100 Washtenaw Ave., Ann Arbor, MI 48109
						<br>
					</p>
				</div>
			</div>

			<div class="col-md-3 widget">
				<h3 class="widget-title">
					Follow me<br>
					关注我
				</h3>
				<div class="widget-body">
					<p class="follow-me-icons">
						<a href="https://www.linkedin.com/in/fan-feng-974842120/"><i class="fa fa-linkedin fa-2"></i></a>
						<a href="https://scholar.google.com/citations?user=1EXo-hIAAAAJ&hl=en"><i class="fa fa-google-plus-square fa-2"></i></a>
						<a href="https://github.com/dmcbffeng"><i class="fa fa-github fa-2"></i></a>
						<a href="https://www.facebook.com/fan.feng.79274"><i class="fa fa-facebook fa-2"></i></a>
					</p>
				</div>
			</div>

			<div class="col-md-3 widget">
				<h3 class="widget-title">
					Thanks for visiting<br>
					感谢来访！
				</h3>
				<div class="widget-body">
					<p> </p>
				</div>
			</div>

		</div> <!-- /row of widgets -->
	</div>
</footer>

<footer id="underfooter">
	<div class="container">
		<div class="row">

			<div class="col-md-6 widget">
				<div class="widget-body">
					<p>Template: Initio by GetTemplate</p>
				</div>
			</div>

		</div> <!-- /row of widgets -->
	</div>
</footer>



<!-- JavaScript libs are placed at the end of the document so the pages load faster -->
<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
<script src="http://netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.min.js"></script>
<script src="assets/js/template.js"></script>
</body>
</html>
